addon/styles/addon.scss

```diff
@@ -27,7 +27,7 @@ $info-bar-title-margin-bottom: 8px;
     }
 
     .sub-title {
-      font-size: $frost-font-s;
+      font-size: $frost-font-xs;
```

Comment:
I don't think this was the intention that Marty noted (he wanted larger) - has this gone past UX and if so can you send me a screenshot from you/Brian?